### DISCUSSION TITLE
[Feature Request] Seperate beta/demo versions from release versions

### DISCUSSION THREAD

**Discussion by TorutheRedFox** [LBPUnion/ProjectLighthouse#425] [Ideas]
An option to have certain title ID have their own environments would be great for the LBP betas, as for example, LBP2 Beta levels and earth decorations don't load in the final game and vice versa.

**Comment by TorutheRedFox** [comment]
basically allowing server owner to make levels and planet decorations published by a user on one of a few certain title IDs (such as NPUA70117) to be exclusive to those title IDs, and allow selective excluding for other title ID groups so that say you have LBP1 levels in LBP2 beta, but not final LBP2 levels, and all levels in final LBP2 except LBP2 beta levels (as they don't work with each other)

**Comment by jvyden** [comment]
Honestly I think the real solution is to treat these versions as separate versions entirely. So for example LBP2 Beta and LBP2 are different entries in `GameVersion.cs`, and it's in-between LBP1 and LBP2. 

That way beta players don't see levels from final, and it would fix this too.

**Comment by TorutheRedFox** [comment]
and exclude beta levels from final too as they don't load

**Comment by Toastbrot236** [comment]
that last one sounds like cap

**Comment by TorutheRedFox** [comment]
they literally don't

I tried

**Comment by TorutheRedFox** [comment]
beta adds &beta=1 to the login request

**Comment by TorutheRedFox** [comment]
ok so about the backwards compatibility, I had my LBP2 eboot set up wrong lol

**Reply by TorutheRedFox** [reply]
it is still very picky though
